<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QC Events</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="open_db_button">
          <property name="text">
           <string>Open Database file for Network</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_cat_button">
          <property name="text">
           <string>Open Earthquake Catalogue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_xml_button">
          <property name="text">
           <string>Open StationXML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ref_radioButton">
          <property name="text">
           <string>Get Reference Station Data</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTreeWidget" name="station_view">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWebView" name="web_view" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="url" stdset="0">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_upd_xml_sql"/>
    <addaction name="action_generate_sql"/>
    <addaction name="action_get_gaps_sql"/>
    <addaction name="action_plot_gaps_overlaps"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_upd_xml_sql">
   <property name="text">
    <string>Upd StnXML frm SQL</string>
   </property>
  </action>
  <action name="action_generate_sql">
   <property name="text">
    <string>Create SQLite DB</string>
   </property>
  </action>
  <action name="action_get_gaps_sql">
   <property name="text">
    <string>Get Gap Info frm SQL</string>
   </property>
  </action>
  <action name="action_plot_gaps_overlaps">
   <property name="text">
    <string>Plot Gaps/Overlaps</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
